src/fitnesse/ContextConfigurator.java

```diff
@@ -66,6 +67,7 @@ public static ContextConfigurator empty() {
   public static ContextConfigurator systemDefaults() {
     return empty()
       .withRootPath(DEFAULT_PATH)
+      .withClassLoader(ClassLoader.getSystemClassLoader())
```

Comment:
I just figured out that this is the class loader used when created for jUnit test. This means the class loader can no longer be changed for jUnit tests by a runner (which is what `FitNesseRunnerExtensionTest.createContext()` is trying to demonstrate).

Can we either add a method `ClassUtils.getClassLoader()` and use that here instead of directly using the system one, or provide some what to supply a class loader from the jUnit runner which subclasses could update, so the class loader created by `PluginsClassLoader.loadPlugins()` is used